packages/taquito-local-forging/src/validator.ts

```diff
@@ -1,3 +1,5 @@
+import { OpKind } from './../../taquito-rpc/src/opkind';
```

Comment:
The import should be `import { OpKind } from '@taquito/rpc'` because rpc and local-forging are distinct packages